server/streaming/streaming.go

```diff
@@ -62,8 +63,8 @@ type Server interface {
 
 // Runtime is the interface to execute the commands and provide the streams.
 type Runtime interface {
-	Exec(containerID string, cmd []string, in io.Reader, out, err io.WriteCloser, tty bool, resize <-chan remotecommand.TerminalSize) error
-	Attach(containerID string, in io.Reader, out, err io.WriteCloser, tty bool, resize <-chan remotecommand.TerminalSize) error
+	Exec(ctx context.Context, containerID string, cmd []string, in io.Reader, out, err io.WriteCloser, tty bool, resize <-chan remotecommand.TerminalSize) error
```

Comment:
while fixing lint in [suite_test.go](https://github.com/cri-o/cri-o/blob/master/server/suite_test.go#L173), this change has a compatibility issue with kubelet interface https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cri/streaming/server.go#L63 as it doesn't pass ctx object, how would like to fix this ?

Reply:
damn yeah we'll have to drop them here and store them for this...